server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -68,6 +68,7 @@
 import org.opensearch.common.io.stream.BytesStreamOutput;
 import org.opensearch.common.lease.Releasable;
 import org.opensearch.common.lifecycle.AbstractLifecycleComponent;
+import org.opensearch.common.lucene.index.OpenSearchDirectoryReader;
```

Comment:
nit: might be better to import `DelegatingCacheHelper` since that is the only usage of `OpenSearchDirectoryReader`